ktor-server/ktor-server-core/jvm/src/io/ktor/routing/RoutingResolve.kt

```diff
@@ -23,8 +23,16 @@ public sealed class RoutingResolveResult(public val route: Route) {
     /**
      * Represents a successful result
      */
-    public class Success(route: Route, override val parameters: Parameters) : RoutingResolveResult(route) {
-        override fun toString(): String = "SUCCESS${if (parameters.isEmpty()) "" else "; $parameters"} @ $route)"
+    public class Success internal constructor(
+        route: Route,
+        override val parameters: Parameters,
+        internal val quality: Double
+    ) : RoutingResolveResult(route) {
+
+        @Deprecated("Binary compatibility")
```

Comment:
Fix comment to describe that it will become internal and instantiation this class will be no longer allowed